test/e2e/load_balancer.go

```diff
@@ -38,7 +38,7 @@ import (
 var _ = Describe("Service [Slow]", func() {
 	f := framework.NewDefaultFramework("service")
 
-	It("should be possible to create and mutate a Service type:LoadBalancer", func() {
+	It("[Canary] should be possible to create and mutate a Service type:LoadBalancer", func() {
```

Comment:
This should be the only change made to the actual CCM code.

Reply:
I am under the assumption  that for canary testing we just wanted a simple test that exercised the load-balancer functionality.  We then began by considering whether we should write a simple python based tests (like the volume-provisioner and flexvolume-driver) or use the existing e2e tests. People voted on and chose the later. So in order to be able to let us easily 'focus' on a single load-balancer test  I added this marker to the description as that seemed to be the way ginkgo, the kubernetes e2e tests, sonobuoy, etc. handle it. I was expecting if we want to include other part of the suite we would annotate them in a similar way.

Reply:
I moved the tag to end of the test message to be more in-line with the K8s e2e tests. e.g. [K8s e2e conformance tests](https://github.com/kubernetes/kubernetes/blob/28b6fb5f7d4ec9fc0be2843d61543ef70b748e70/test/e2e/framework/framework.go#L698:6)